danceschool/vouchers/helpers.py

```diff
@@ -59,10 +67,10 @@ def createReferrerVoucher(customer):
     forFirstTimeCustomersOnly = False
     expirationDate = None
     disabled = False
-    name = _("Referral Bonus for %s, %s" % (customer.fullName,customer.email))
+    name = _("Referral Bonus for %s, %s" % (customer.fullName, customer.email))

```

Comment:
This will never be translated, as the string passed to `_` will not contain the `%s` at run-time.

```suggestion
    name = _("Referral Bonus for %s, %s") % (customer.fullName, customer.email)
```

Reply:
@merwok: Good catch, added in the missing parenthesis to address your comment. Thanks!

Reply:
I think you added an extra paren :)

Reply:
Whoops. Extra paren removed.